duties/cli/arguments.py

```diff
@@ -22,10 +22,12 @@ def __get_raw_arguments() -> Namespace:
         usage="eth-duties [...options]",
     )
     parser.add_argument(
-        "--beacon-node",
+        "--beacon-nodes",
         type=parse.set_beacon_node_url,
-        help="URL to access the beacon node api (default: http://localhost:5052)",
-        action="store",
+        help=(
+            "Comma separated list of URLs to access the beacon node api "
```

Comment:
lint line 28. extra space at the end.

Reply:
Do you mean the extra space after `api`? This is intended since it is a multiline string. So the space is to separate this line and the next :slightly_smiling_face: